guides/api-security.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: API security
+description: Learn how to secure your web applications and APIs with Architect
```

Comment:
We have removed all the descriptions because they appear in a different font on the page. Lindsay asked me if we could change that.